documentation/workflows.md

```diff
@@ -0,0 +1,74 @@
+# Working on the Project
+
+Better the be explicit/specific on topics than to wait for avoidable confusion due to ambiguity.
+
+## Principles
+
+1. No work is ever done directly on `main`; ALWAYS branch off of the target receiving branch (i.e. `main` for the majority of work)
+2. All changes must be reviewed and approved, and pass all quality gates
```

Comment:
I think some very concrete information here would be useful -- who are the expected reviewers? in what time frame will review happen? what are the quality gates (coverage expectations, maybe? code style guidelines?)

Reply:
Check back and let me know if what I added is helpful, or if more is needed.